openhcl/virt_mshv_vtl/src/processor/tdx/mod.rs

```diff
@@ -1870,6 +1890,14 @@ impl UhProcessor<'_, TdxBacked> {
                 }
                 &mut self.backing.vtls[intercepted_vtl].exit_stats.tdcall
             }
+            VmxExit::EXCEPTION => {
+                tracing::info!(
```

Comment:
downgrade this trace - info should never be used for things that can happen normally at steady state. I'd probably make this `trace` level since it probably would be quite spammy.